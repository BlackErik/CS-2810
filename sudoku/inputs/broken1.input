.17..5.93...4...6.4....9..2.6...7..........18..58..6..72.......9....2...3..6...87
