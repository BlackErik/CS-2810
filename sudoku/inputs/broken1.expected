Please enter an 81-character board:

Board with up-to-date pencil marks:
+-----------------------+-----------------------+-----------------------+
|       |       |       |       |       |       |       |       |       |
|   6       1       7   |   2       8       5   |   4       9       3   |
|       |       |       |       |       |       |       |       |       |
| - - - + - - - + - - - | - - - + - - - + - - - | - - - + - - - + - - - |
|       |       |       |       |       |       |       |       |       |
|   5       9       2   |   4       7       3   |   8       6       1   |
|       |       |       |       |       |       |       |       |       |
| - - - + - - - + - - - | - - - + - - - + - - - | - - - + - - - + - - - |
|       |       |       |       |       |       | . . . | . . . |       |
|   4       3       8   |   1       6       9   | . 5 .   . 5 .     2   |
|       |       |       |       |       |       | 7 . . | 7 . . |       |
+-----------------------+-----------------------+-----------------------+
|       |       | . . 3 | . . 3 | 1 2 3 |       | . . 3 | . 2 3 | . . . |
|   8       6     . . . | . 5 .   . 5 .     7   | . 5 .   4 5 .   4 5 . |
|       |       | . . 9 | . . 9 | . . . |       | . . . | . . . | . . . |
| - - - + - - - + - - - | - - - + - - - + - - - | - - - + - - - + - - - |
|       |       | . . 3 | . . 3 | . . 3 |       | . . 3 |       |       |
|   2       4     . . . | . 5 .   . 5 .     6   | . 5 .     1       8   |
|       |       | . . 9 | . . 9 | . . . |       | 7 . . |       |       |
| - - - + - - - + - - - | - - - + - - - + - - - | - - - + - - - + - - - |
|       |       |       |       | . 2 3 |       |       | . 2 3 |       |
|   1       7       5   |   8     . . .     4   |   6     . . .     9   |
|       |       |       |       | . . . |       |       | . . . |       |
+-----------------------+-----------------------+-----------------------+
|       |       | 1 . . | . . 3 | . . 3 |       | 1 . 3 | . . 3 | . . . |
|   7       2     . . 6 | . 5 .   4 5 .     8   | . 5 .   4 5 .   4 5 6 |
|       |       | . . . | . . . | . . . |       | . . 9 | . . . | . . . |
| - - - + - - - + - - - | - - - + - - - + - - - | - - - + - - - + - - - |
|       |       | 1 . . | . . 3 | . . 3 |       | 1 . 3 | . . 3 | . . . |
|   9       8     . . 6 | . 5 .   4 5 .     2   | . 5 .   4 5 .   4 5 6 |
|       |       | . . . | 7 . . | . . . |       | . . . | . . . | . . . |
| - - - + - - - + - - - | - - - + - - - + - - - | - - - + - - - + - - - |
|       |       |       |       |       |       |       |       |       |
|   3       5       4   |   6       9       1   |   2       8       7   |
|       |       |       |       |       |       |       |       |       |
+-----------------------+-----------------------+-----------------------+

Board after solving naked sets:
+-----------------------+-----------------------+-----------------------+
|       |       |       |       |       |       |       |       |       |
|   6       1       7   |   2       8       5   |   4       9       3   |
|       |       |       |       |       |       |       |       |       |
| - - - + - - - + - - - | - - - + - - - + - - - | - - - + - - - + - - - |
|       |       |       |       |       |       |       |       |       |
|   5       9       2   |   4       7       3   |   8       6       1   |
|       |       |       |       |       |       |       |       |       |
| - - - + - - - + - - - | - - - + - - - + - - - | - - - + - - - + - - - |
|       |       |       |       |       |       |       |       |       |
|   4       3       8   |   1       6       9   |   5       7       2   |
|       |       |       |       |       |       |       |       |       |
+-----------------------+-----------------------+-----------------------+
|       |       |       |       |       |       |       |       |       |
|   8       6       9   |   5       1       7   |   3       2       4   |
|       |       |       |       |       |       |       |       |       |
| - - - + - - - + - - - | - - - + - - - + - - - | - - - + - - - + - - - |
|       |       |       |       | . . . |       |       |       |       |
|   2       4       3   |   9     . . .     6   |   7       1       8   |
|       |       |       |       | . . . |       |       |       |       |
| - - - + - - - + - - - | - - - + - - - + - - - | - - - + - - - + - - - |
|       |       |       |       |       |       |       | . . . |       |
|   1       7       5   |   8       2       4   |   6     . . .     9   |
|       |       |       |       |       |       |       | . . . |       |
+-----------------------+-----------------------+-----------------------+
|       |       |       |       | . . . |       |       | . . . |       |
|   7       2       1   |   3     4 5 .     8   |   9     4 5 .     6   |
|       |       |       |       | . . . |       |       | . . . |       |
| - - - + - - - + - - - | - - - + - - - + - - - | - - - + - - - + - - - |
|       |       |       |       | . . . |       |       | . . . | . . . |
|   9       8       6   |   7     . . .     2   |   1     . . .   . . . |
|       |       |       |       | . . . |       |       | . . . | . . . |
| - - - + - - - + - - - | - - - + - - - + - - - | - - - + - - - + - - - |
|       |       |       |       |       |       |       |       |       |
|   3       5       4   |   6       9       1   |   2       8       7   |
|       |       |       |       |       |       |       |       |       |
+-----------------------+-----------------------+-----------------------+

Board is unsolvable
